app/src/main/java/org/fossasia/phimpme/editor/fragment/CropFragment.java

```diff
@@ -39,11 +39,13 @@
   private LinearLayout ratioList, imageList, combinedList;
   private static List<RatioItem> dataList = new ArrayList<RatioItem>();
```

Comment:
even here xD

Reply:
I'll leave this one. This one wasn't done by me. It's been there from the beginning.

Reply:
Ok, No worries